core/src/impl/Kokkos_Profiling_Interface.hpp

```diff
@@ -86,126 +84,43 @@ inline uint32_t device_id(ExecutionSpace const& space) noexcept {
 #if defined(KOKKOS_ENABLE_PROFILING)
 // We check at configure time that libdl is available.
 #include <dlfcn.h>
+#endif
 
 #include <impl/Kokkos_Profiling_DeviceInfo.hpp>
-
-#define KOKKOSP_INTERFACE_VERSION 20171029
+#include <impl/Kokkos_Profiling_DeviceInfo.hpp>
+#include <impl/Kokkos_Profiling_C_Interface.h>
+#include <impl/Kokkos_Profiling_C_Interface.h>
```

Comment:
Included twice

Reply:
Weird. Fixed

Reply:
https://github.com/kokkos/kokkos/blob/a11bdf3a0e80ce4a7dd6af87a6b4e7a9bd000464/core/src/impl/Kokkos_Profiling_Interface.hpp#L89-L90

Reply:
Yikes, thanks. Not sure how that happened